application/src/main/java/org/togetherjava/tjbot/commands/github/GitHubCommand.java

```diff
@@ -0,0 +1,121 @@
+package org.togetherjava.tjbot.commands.github;
+
+import net.dv8tion.jda.api.events.interaction.command.CommandAutoCompleteInteractionEvent;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import org.kohsuke.github.GHIssue;
+import org.kohsuke.github.GHIssueState;
+
+import org.togetherjava.tjbot.features.CommandVisibility;
+import org.togetherjava.tjbot.features.SlashCommandAdapter;
+import org.togetherjava.tjbot.features.utils.StringDistances;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.*;
+import java.util.function.ToIntFunction;
+import java.util.regex.Matcher;
+import java.util.stream.Stream;
+
+/**
+ * Slash command (/github) used to search for an issue in one of the repositories listed in the
+ * config. It even comes with auto completion!
+ */
+public class GitHubCommand extends SlashCommandAdapter {
+    private static final Duration CACHE_EXPIRES_AFTER = Duration.ofMinutes(1);
+
+    /**
+     * Compares the getUpdatedAt values.
```

Comment:
ascending or descending and what? reformulate `Compares two GitHub Issues ascending by the time they have been updated at.`